build_and_push.sh

```diff
@@ -15,13 +15,15 @@ function echo_run() {
     "$@"
 }
 
-# default REMOTE
+# default argument values
 REMOTE="docker.io/finkandreas"
+CONFIG_FILE="build-versions.yaml"
 
-while getopts "lhr:" Option
+while getopts "lhrf:" Option
 do
   case $Option in
     r     ) REMOTE=${OPTARG};;
+    f     ) CONFIG_FILE=${OPTARG};;
```

Comment:
Although I have suggested `-f config_file`, I now have the feeling that `-c config_file` is more appropriate? I don't know. Both ok, but I think `-c config_file` is more common.

Reply:
I agree